src/cli/setup_mode.rs

```diff
@@ -1,5 +1,7 @@
 use anyhow::Result;
-use clap::{builder::PossibleValuesParser, value_parser, Arg, ArgAction, Command};
+use clap::{
+    builder::PossibleValuesParser, error::ErrorKind, value_parser, Arg, ArgAction, Command,
```

Comment:
This too seems unrelated.

Reply:
error::ErrorKind is moved from within main() to a file level import. I don't recall if I had to do that do deal with a transient state during the refactoring, or whether I could move it back now. That said, I usually prefer top of module imports, so we can break if out if you like, but I'm not super excited by it :)

Reply:
I prefer top of module imports too, but when reviewing a large change I prefer for these not to be intertwined with the other changes.